python_src/src/lamp_py/postgres/postgres_utils.py

```diff
@@ -201,13 +195,7 @@ class DatabaseManager:
     manager class for rds application operations
     """
 
-    def __init__(
-        self,
-        verbose: bool = False,
-        seed: bool = False,
-        clear_rt: bool = False,
-        clear_static: bool = False,
-    ):
+    def __init__(self, verbose: bool = False):
```

Comment:
database manager no longer initializes by modifying the db its connecting to. that is handled by another script that uses its functionality.